script/fs-bench/fio/config/randread-4.conf

```diff
@@ -0,0 +1,9 @@
+[randread-4]
+rw=randread
+bs=4k
+size=512M
+numjobs=4
+filename=/work/data/512mib.0
```

Comment:
Is this file `/work/data/512mib.0` contained in the testing stargz image(otherwise, no HTTP requests will occur)?

Reply:
It is containerd, check script/fs-bench/fio/Dockerfile